src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -27,7 +27,8 @@ interface IOptionSettlementEngine {
     error OptionsChainExists(bytes32 hash);
```

Comment:
this error name should change also

Reply:
To what?

Reply:
sry, `OptionsTypeExists`.

Reply:
Merged in master, which had that already.